pysparkling/sql/tests/test_session.py

```diff
@@ -2,11 +2,11 @@
 
 import pytest
 
+from pysparkling import Context
 from pysparkling import StorageLevel
-from pysparkling.sql.types import Row, StructType, StructField, LongType, StringType, DoubleType, \
-    ArrayType, MapType, IntegerType, row_from_keyed_values
 from pysparkling.sql.session import SparkSession
-from pysparkling import Context
+from pysparkling.sql.types import ArrayType, DoubleType, IntegerType, LongType, MapType, Row, row_from_keyed_values, \
+    StringType, StructField, StructType
```

Comment:
Import order is not case sensitive, is it on purpose?

On this line it may be cleaner to have `row_from_keyed_values` before or after all types, and a case sensitiveness would ease that

it would also make sense to have the same for `Row`, but it would not be solve in the same way

Reply:
This is likely my optimize imports taking over :-). It's not case sensitive as far as I know...